chart/k8gb/templates/coredns/cm.yaml

```
@@ -27,6 +27,12 @@ data:
             filter k8gb.absa.oss/dnstype=local
             negttl {{ .dnsZoneNegTTL | default 30 }}
             loadbalance weight
+            {{- if and .geoDataFilePath (ne .geoDataFilePath "") }}
```

```suggestion
            {{- if .geoDataFilePath }}
```
The check for empty string is not needed, it is already evaluated to false:
https://helm-playground.com/#t=LQhQG92ACBLAzaA6AagQwDYFcCmBnJAcxwHsARNAFzQDFYMcAFKgC2gF93Rjyrb6mraJGTps%2BIqQrU6DZpTacIUaDgB2AEw7sgA&v=LQhQHMFMHsBEEMAu8BiBLANpACkgFgFwAEARCaEA

---

Thanks @abaguas, updated